.dev_scripts/benchmark_regression_cfg.yaml

```diff
@@ -1,143 +1,215 @@
 model_list:
-    inference:
-        -   # necessary parameters
-            config: configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/coco/hrnet_w32_coco_256x192.py  # path to the config file
-            checkpoint: https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w32_coco_256x192-c78dce93_20200708.pth # the config file
-            priority: 0 # the priority of the model, 0: core, 1: important, 2: less important, 3: least important
-
-            # optional parameters, if not specified, use the default setting in the .dev_scripts/benchmark_regression.py
-            port: 29666 # the port number used to communicate between different GPUs
-            gpus: 2 # specify the number of GPUs needed according to the specific task
-            gpus_per_node: 2 # the number of GPUs used each computing node
-            cpus_per_task: 5 # the number of cpus per task
-            partition: openmmlab  # the partition name
-            task_name: infer_hrnet_w32_coco_256x192 # the job name shown in slurm
-            py_args: # other arguments to run the script tools/test.py
-                out: results_hrnet_w32_coco_256x192.json  # output result file
-                eval: mAP # evaluation metric, which depends on the dataset, e.g., "mAP" for MSCOCO
-                fuse-conv-bn:
-                gpu_collect:
+    P0: # the priority of the models, P0: core, P1: important, P2: less important, P3: least important
```

Comment:
A few suggestions about the config file:
- Rename this file "benchmark_regression_cfg_tmpl.yaml" or something, which serves as a template to show the full content that a config file could include. We will add a more compact config file with a full model list and only necessary arguments.
- Use `test` instead of `infer` as the mode name.
- `gpus_per_node` can be set to 8 for all models and modes.

Reply:
Got these suggestions.